index.js

```diff
@@ -281,6 +281,11 @@ class ServerlessDynamodbLocal {
         const resources = _.get(stack, "Resources", []);
         return Object.keys(resources).map((key) => {
             if (resources[key].Type === "AWS::DynamoDB::Table") {
+                if (!resources[key].Properties.TableName) {
+                    const service = this.service.service;
+                    const stage = this.options.stage || this.service.provider.stage;
+                    resources[key].Properties.TableName = `${service}-${stage}-${key}`;
```

Comment:
By adding the service name to the TableName it can pretty easily reach the 256 char max length that DynamoDB has for its name. What about limiting the name to stage-key?
```suggestion
                    resources[key].Properties.TableName = `${stage}-${key}`;
```